src/graphorm/index.ts

```diff
@@ -74,6 +76,15 @@ const obj = new GraphORM({
       notificationEmail: {
         transform: nullIfNotSameUser,
       },
+      isTeamMember: {
+        select: (_, alias, qb) =>
+          qb
+            .select('count(*)')
+            .from(Feature, 'f')
+            .where(`f."userId" = ${alias}.id`)
+            .andWhere(`f."feature" = :feature`, { feature: FeatureType.Team }),
```

Comment:
We should be able to use `EXISTS` it will be more efficient then count. 

```suggestion
        select: (_, alias, qb) => {
          const query = qb
            .select('1')
            .from(Feature, 'f')
            .where(`f."userId" = ${alias}.id`)
            .andWhere(`f."feature" = :feature`, { feature: FeatureType.Team });

          return `EXISTS${query.getQuery()}`;
        },
```